ninja/__main__.py

```diff
@@ -0,0 +1,3 @@
+from ninja import ninja
+
+ninja()
```

Comment:
To avoid import-time side-effects this should be:

```suggestion
if __name__ == '__main__':
    ninja()
```